src/octonode/client.coffee

```diff
@@ -142,37 +144,106 @@ class Client
         body = JSON.parse(body || '{}')
       catch err
         return callback(err)
-
+    debugger
     return callback(new HttpError(body.message, res.statusCode, res.headers, body)) if body.message and res.statusCode in [400, 401, 403, 404, 410, 422]
     callback null, res.statusCode, body, res.headers
 
+  errorHandlePromise: (response, cb) ->
+    debugger
+    {statusCode, body, headers} = response
```

Comment:
Can do this destructuring where the arguments are passed in to the function since `response` isn't used anywhere